### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by vanessaholland**

### CODE REVIEW

File: speechAssets/SampleUtterances.txt [hunk 1]
```diff
@@ -0,0 +1,40 @@
+Unhandled I don't know
```
Comment (maulin) [inline speechAssets/SampleUtterances.txt]: do examples need to be checked in?

File: test/events/getAnswerIntent.request.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
```
Comment (maulin) [inline test/events/getAnswerIntent.request.json]: POST to `/brickFTP/permissions`

Comment (maulin) [inline test/events/getAnswerIntent.request.json]: where are these test events used

File: speechAssets/SampleUtterances.txt (hunk 1 above)
Comment (vanessaholland) [inline speechAssets/SampleUtterances.txt]: yes, this is what alexa uses to determine which intent to call based on user speech

File: test/events/getAnswerIntent.request.json (hunk 2 above)
Comment (vanessaholland) [inline test/events/getAnswerIntent.request.json]: by the test/index.js

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+      if (responseData == null) {
+        output = constants.dataErrorMessage;
+      }
+      else {
+        output = constants.questionIntroMessage;
+
+        for (var i = 0; i < numberOfResults; i++) {
+          var title = responseData[i].title;
+          var answer = responseData[i].answer;
+          answer.replace(/Tags:[^<]*/g, '');
```
Comment (maulin) [inline src/index.js]: `answer.gsub(/Tags:[^<]*/, '')`

**Pull request closed without merging**